src/asn1acn.h

```diff
@@ -81,8 +87,12 @@ private slots:
     void addToToolsMenu(ActionContainer *container);
 
     QPointer<Libraries::Wizard::ImportComponentWizard> m_importComponentWizard;
+
```

Comment:
unnecessary \n (or one after the next line is needed :)) it makes "test generator" grouped with "find usages".

Reply:
Will correct it.